Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -97,32 +97,55 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 
 		bool IsObservableCollection(object source)
 		{
-			var type = source.GetType();
-			return type.IsGenericType &&
-				   type.GetGenericTypeDefinition() == typeof(ObservableCollection<>);
+			/* #fix=7167
```

Comment:
While I appreciate the effort to clarify things, I feel it might be a bit too much. This comment I think speaks for itself in code.

Reply:
Thank you for the review. Should I delete the comments in the code which you pinned?

Reply:
Maybe let's see what the other reviewer will think about it. But as far as I am concerned they might go yes.

Also, did you sign the CLA yet? (See the first comment) it seems you still need to do that :)

Reply:
Ok thanks. No i did not sign the CLA yet.
I will be able to sign it tomorrow or the other day.